app/controllers/course/condition/levels_controller.rb

```diff
@@ -0,0 +1,42 @@
+class Course::Condition::LevelsController < Course::ConditionsController
+  load_resource :level_condition, class: Course::Condition::Level.name, parent: false
+
+  def new
+  end
+
+  def create
+    @level_condition.acting_as.conditional = @conditional
+    @level_condition.acting_as.course = @course
```

Comment:
My understanding is that `before_action` is used for repeated tasks that occur in many action. In this case, we only need to set these values before creating the model. Why is it helpful to keep it in `before_action`?